riscv/csrs.cc

```diff
@@ -1312,7 +1314,6 @@ reg_t dcsr_csr_t::read() const noexcept {
   result = set_field(result, DCSR_EBREAKU, ebreaku);
   result = set_field(result, CSR_DCSR_EBREAKVS, ebreakvs);
   result = set_field(result, CSR_DCSR_EBREAKVU, ebreakvu);
-  result = set_field(result, DCSR_STOPCYCLE, 0);
```

Comment:
Can you explain this change?

Reply:
The DCSR layout in riscv-opcodes was updated to match the latest definition in debug 1.0 specification. The latest debug 1.0 definition does not have a STOPCYCLE or HALT field in dcsr. This was deleted since these macros no longer exist in opcodes.h

Reply:
Got it. Can you split these two changes into a separate first commit, and add the other changes as a subsequent second commit?